newrelic/structures_newrelic_synthetics_simple_browser_monitor.go

```diff
@@ -148,17 +155,19 @@ func buildSyntheticsSimpleBrowserMonitorUpdateStruct(d *schema.ResourceData) syn
 	do, doOk := d.GetOk("device_orientation")
 	dt, dtOk := d.GetOk("device_type")
 
-	if doOk || dtOk {
-		simpleBrowserMonitorUpdateInput.AdvancedOptions.DeviceEmulation = &synthetics.SyntheticsDeviceEmulationInput{}
-
-		if doOk {
-			simpleBrowserMonitorUpdateInput.AdvancedOptions.DeviceEmulation.DeviceOrientation = synthetics.SyntheticsDeviceOrientation(do.(string))
-		}
+	if !(scriptLangOk && runtimeTypeOk && runtimeTypeVersionOk) && (doOk && dtOk) {
+		return simpleBrowserMonitorUpdateInput, errors.New("device emulation is not supported by legacy runtime")
+	}
```

Comment:
`scriptLangOk` might need to be excluded from this condition.

I find that specifying both `runtime_type` and `runtime_version` without `script_language` in the configuration does not lead to an error (the Synthetics API seems to default an unspecified `scriptLanguage` to `JAVASCRIPT` in the backend) and hence, we don't get a `Bad Runtime Values` error from the Backend like we do if `runtime_type` is specified and `runtime_version` is not; hence the above comment.